Documentation/api.md

```diff
@@ -243,6 +243,7 @@ The `NodeExporterCollectorNetClassConfig` resource works as an on/off switch for
 | -------- | ---- | ----------- |
 | enabled | bool | A Boolean flag that enables or disables the `netclass` collector. |
 | useNetlink | bool | A Boolean flag that activates the `netlink` implementation of the `netclass` collector. This implementation improves the performance of the `netclass` collector by omitting these metrics: `node_network_address_assign_type`, `node_network_name_assign_type`, `node_network_device_id`, `node_network_speed_bytes`. In addition, The `node_network_info` metric lacks the `duplex` label. |
+| ignoredNetworkDevices | *[]string | A list of network devices to be excluded from collection, using regular expressions. By default, the following list is used: `veth.*`, `[a-f0-9]{15}`, `enP.*`, `ovn-k8s-mp[0-9]*`, `br-ex`, `br-int`, `br-ext`, `br[0-9]*`, `tun[0-9]*`, `cali[a-f0-9]*` |
```

Comment:
Should I add some kind of warning here, that removing filters from the default list will increase cardinality?

Reply:
Especially for filters such as `[a-f0-9]{15}` or `cali[a-f0-9]*` , removal must be considered with care